use failure::Error;

use sento_api::run_server;

fn main() -> Result<(), Error> {
    run_server()
}
